dashboard/src/components/MessageDetails/MessageDetails.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import * as Modal from "react-modal";
+
+import "./MessageDetails.css";
+
+interface IMessageDetailsProps {
+  modalIsOpen: boolean;
+  closeModal: () => Promise<any>;
+  message: string;
+}
+
+class MessageDetails extends React.Component<IMessageDetailsProps> {
```

Comment:
Could you make this component `Pure`?

Reply:
_agreed offline to make this a Stateless component_